download.sh

```diff
@@ -1,24 +0,0 @@
-#!/usr/bin/env bash
```

Comment:
we want to leave download.sh, setup.sh, etc. in place for the standard mac os setup right?